docs/source/pythonapi/examples/mgxs-part-i.ipynb

```diff
@@ -794,19 +795,19 @@
        "  <tbody>\n",
        "    <tr>\n",
        "      <th>1</th>\n",
-       "      <td>1</td>\n",
-       "      <td>1</td>\n",
-       "      <td>total</td>\n",
-       "      <td>0.668323</td>\n",
-       "      <td>0.001264</td>\n",
+       "      <td> 1</td>\n",
+       "      <td> 1</td>\n",
+       "      <td> total</td>\n",
+       "      <td> 0.668323</td>\n",
+       "      <td> 0.001264</td>\n",
        "    </tr>\n",
        "    <tr>\n",
        "      <th>0</th>\n",
-       "      <td>1</td>\n",
-       "      <td>2</td>\n",
-       "      <td>total</td>\n",
-       "      <td>1.293258</td>\n",
-       "      <td>0.007624</td>\n",
+       "      <td> 1</td>\n",
```

Comment:
I don't know why there is new whitespace here.  Maybe it's some formatting change with my version of Jupyter?